src/EditorFeatures/CSharp/FixInterpolatedVerbatimString/FixInterpolatedVerbatimStringCommandHandler.cs

```diff
@@ -0,0 +1,97 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.ComponentModel.Composition;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.Editor.Shared.Extensions;
+using Microsoft.CodeAnalysis.Editor.Shared.Utilities;
+using Microsoft.CodeAnalysis.Text;
+using Microsoft.VisualStudio.Commanding;
+using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.Text.Editor.Commanding.Commands;
+using Microsoft.VisualStudio.Text.Operations;
+using Microsoft.VisualStudio.Utilities;
+using VSCommanding = Microsoft.VisualStudio.Commanding;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.FixInterpolatedVerbatimString
+{
+    /// <summary>
+    /// Replaces <c>@$"</c> with <c>$@"</c>, which is the preferred and until C# 8.0 the only supported form
+    /// of an interpolated verbatim string start token
```

Comment:
Should this comment that even in the 8.0 case, we are still preferring this for consistency? Otherwise the lack of langversion checks in this looks conspicuous in the face of this comment, even though I think it's the right thing to do.

Reply:
The comment says this is `preferred` before commenting this is unsupported until C# 8.0, but I will clarify this a bit more.

Reply:
➡️ Fixed in 0407e108dcfba01a0d8a9f153188d425cfde73a1